docs/github-action.md

```diff
@@ -0,0 +1,85 @@
+
+## Scheduled scans
+
+Regularly scanning your project for vulnerabilities can alert you to new vulnerabilities in your dependency tree. This GitHub Action will scan your project on a set schedule and report all known vulnerabilities. 
+
+### Instructions
+
+In your project repository, create a new file `.github/workflows/osv-scanner-scheduled.yml`. 
+
+Include the following in the [`osv-scanner-scheduled.yml`](https://github.com/google/osv-scanner/blob/main/.github/workflows/osv-scanner-scheduled.yml) file:
+
+```yml
+name: OSV-Scanner Scheduled Scan
+
+on:
+  schedule:
+    - cron: '12 12 * * 1'
+  push:
+    branches: [ "main" ]
+
+permissions: 
+  # Require writing security events to upload SARIF file to security tab
+  security-events: write
+  # Only need to read contents
+  contents: read
+
+jobs:
+  scan-scheduled:
+    uses: "google/osv-scanner/.github/workflows/osv-scanner-reusable-scheduled.yml@main"
+```
+The above code assumes that your primary branch is called `main`, if you use another name, like `master`, you will need to update the file to correspond with your primary branch name.
+
+As written, the scanner will run on 12:12 pm UTC every Monday. You can change the schedule by following the instructions [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule). 
+
+### View results
+
+Maintainers can review results of the scan by navigating to their project's security > code scanning tab. Vulnerability details can also be viewed by clicking on the details of the failed action.
+
+## Scans on PRs
+
+Scanning your project on each pull request can help you keep vulnerabilities out of your project. This GitHub Action compares a vulnerability scan of the main branch to a vulnerability scan of the feature branch. You will be notified of any new vulnerabilities introduced through the feature branch. You can also choose to [prevent merging](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches#require-status-checks-before-merging) if new vulnerabilities are introduced through the feature branch.
+
+### Instructions
+
+In your project repository, create a new file `.github/workflows/osv-scanner-pr.yml`. 
+
+Include the following in the [`osv-scanner-pr.yml`](https://github.com/google/osv-scanner/blob/main/.github/workflows/osv-scanner-pr.yml) file:
+
+```yml
+name: OSV-Scanner PR Scan
+
+on:
+  pull_request:
+    branches: [ main ]
+  merge_group:
+    branches: [ main ]
+
+# Declare default permissions as read only.
+permissions: read-all
+
+jobs:
+  scan-pr:
+    uses: "google/osv-scanner/.github/workflows/osv-scanner-reusable-pr.yml@main"
+```
+
+The above code assumes that your primary branch is called `main`, if you use another name, like `master`, you will need to update the file to correspond with your primary branch name.
```

Comment:
@another-rex I tried to use this GitHub action on a personal project on mine, just so I could understand the user experience. I couldn't get it to show up as a check that runs on a PR until I realized that the project I was trying to use the action on used `master` instead of `main`. 

I have updated the docs to help people in similar situations, but I don't know if that would have been an obvious fix for people. I eventually figured it out. What do you think?

Reply:
Yeah I think adding a clarification makes sense, maybe instead of having it as a separate text field underneath add it as a comment above [ main ]?

e.g.
```
# Change `main` to your default branch if you have a different default branch
      branches: [ main ]
```